triedb/pathdb/asyncnodebuffer.go

```diff
@@ -284,7 +271,7 @@ func copyNodeCache(n *nodecache) *nodecache {
 	if n == nil || n.buffer == nil {
 		return nil
 	}
-	nc := newNodeCache(int(n.limit), n.nodes, n.states, n.layers)
```

Comment:
follow the code in v1.4.16
https://github.com/bnb-chain/bsc/blob/v1.4.16/triedb/pathdb/asyncnodebuffer.go#L439